src/app/monitoring/containers/live-logs/live-logs.component.scss

```diff
@@ -1,15 +1,43 @@
+@import '../../../core/core';
+
+$text-color: $gray-700;
+$text-color-light: $gray-600;
+
 .container {
-    margin-top: 1%;
+    margin-top: 0.5vh;
     max-width: 100vw;
 }
 .nav-container {
-    display: grid;
-    grid-template-columns: 13% 20% 11%;
-    grid-gap: 1em;
+    display: flex;
+    flex-wrap: wrap;
```

Comment:
Der Container wraped in der Realität nicht, selbst auf der Geräteklasse iPhone 5/ 5s nicht. Und das ist auch gut so also warum setzt du diese Property?

Reply:
um das flex-wrap Verhalten zu untersuchen und zu beurteilen, ob es sich eignet, um die anderen beiden Loggingkomponenten, in denen es auf jeden fall zu Umbrüchen kommt, mit einer reinen flex-Architektur so zu gestalten, wie wir es wollen.